autogluon/utils/tabular/ml/models/fastainn/hyperparameters/searchspaces.py

```diff
@@ -0,0 +1,40 @@
+from .....ml.constants import BINARY, MULTICLASS, REGRESSION
+from .......core import Categorical, Real, Int
+
+
+def get_default_searchspace(problem_type, num_classes=None):
+    if problem_type == BINARY:
+        return get_searchspace_binary().copy()
+    elif problem_type == MULTICLASS:
+        return get_searchspace_multiclass(num_classes=num_classes)
+    elif problem_type == REGRESSION:
+        return get_searchspace_regression().copy()
+    else:
+        return get_searchspace_binary().copy()
+
+
+def get_searchspace_binary():
+    spaces = {
+        # See docs: https://docs.fast.ai/tabular.models.html
+        'layers': Categorical(None, [200, 100], [256], [100, 50], [200, 100, 50], [50, 25], [300, 150]),
```

Comment:
May want more diverse default search space for network architecture, eg. something like:

`Categorical(None, [200, 100], [200], [1000], [1000, 500], [50, 25], [200, 100, 50], [1000, 500, 200])`

Reply:
updated to `Categorical(None, [200, 100], [200], [500], [1000], [500, 200], [50, 25], [1000, 500], [200, 100, 50], [500, 200, 100], [1000, 500, 200])`